src/Service/Core.php

```diff
@@ -121,15 +149,12 @@ public function getConfigFromJson($module, $type = NULL) {
    * Get the current text and configuration settings for an app.
    *
    * @param string $module
-   *   The machine name of the extending module.
+   *   The machine name of the module being installed.
    * @param string $language
    *   The language code.
    *
-   * @return string
-   *   The text and configuration settings for the app, as JSON.
-   *
-   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
-   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   * @return array
+   *   The text and configuration settings for the app json endpoint, as array.
    */
   public function getAppConfig($module, $language) {
     $config = new \StdClass();
```

Comment:
Does this need to be a class obj?
given it is used to store a json array on line 169..

```
$config = new \StdClass();
....
$config = Json::decode($config_json);
```

Reply:
you could make the language param optional and check something like:
```
if ($language == NULL ){
  $language = $this->languageManager->getCurrentLanguage()->getId();
}
```

Reason: for single language sites, it makes integration a little easier. Also, it provides a self explanatory example of how to get language, so its easier for anyone calling the method.